### ISSUE TITLE
Add handle to reload daemons in systemd

### ISSUE THREAD

**Issue by Granddave** [nolte/ansible-minecraft#19]
Hi,
If a change is made resulting in an altered unit file, eg. changing allocated memory usage, systemd wants a daemon-reload (see commands below). Is it possible to add a handle to do this automatically?

```
root@mc-1:~# systemctl restart minecraft.service
Warning: The unit file, source configuration file or drop-ins of minecraft.service changed on disk. Run 'systemctl daemon-reload' to reload units.
root@mc-1:~# systemctl daemon-reload
root@mc-1:~# systemctl restart minecraft.service
```

Thanks

**Comment by nolte** [comment]
Hi, 

First Thanks for the feedback! I added a handler for reload the systemd daemons, if the service changed.
Please check the branch fix/reload-systed-19.
When the changes are ok than i will release the Role, so please give me feedback ;)

Thanks

**Comment by Granddave** [comment]
Thanks, will try it out later today. 👍

**Comment by nolte** [comment]
Fixed with v5.0.2.
